harmony/harmony.py

```diff
@@ -275,13 +295,15 @@ def _cloud_access_url(self) -> str:
 
     def _params(self, request: Request) -> dict:
         """Creates a dictionary of request query parameters from the given request."""
-        params = {'forceAsync': True}
+        params = {'forceAsync': 'true'}
```

Comment:
It would be nice to use a boolean rather than a string for true/false. Any reason this had to change?

Reply:
It's the responsibility of this method to get values into their serialized form.  A few lines down on line 309, you can see where we are explicitly changing passed booleans into lower-case string representations.  This change wasn't technically needed, but the inconsistency and serialization of the value as 'True' instead of 'true' cost me debugging time, so I figured I'd change it.